Automattic/vip-go-mu-plugins
# Search-dev-tools fix linting issues

## Description



## Changelog Description

### Plugin Updated: Search

Fix minor linting issues in search-dev-tools panel.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out the PR
2. Fire up dev-env
3. Click around in the dev tools panel
